src/route/graph/components/graph-components/NodeDetails.tsx

```diff
@@ -223,25 +224,30 @@ const NodeDetails: React.FC<NodeDetailsProps> = ({
             <ListItemIcon>
               <ArrowRightAlt />
             </ListItemIcon>
-            <ListItemText primary={data.id} secondary={'ID'} />
+            <ListItemText
+              primary={
+                <MultiCopy
+                  type="caption"
+                  values={[data.address, data.address6]}
+                />
+              }
+              secondary={t('node.addresses')}
+            />
           </ListItem>
           <ListItem>
             <ListItemIcon>
               <ArrowRightAlt />
             </ListItemIcon>
             <ListItemText
-              primary={data.publickey}
+              primary={<MultiCopy type="subtitle2" values={[data.publickey]} />}
```

Comment:
the above is a caption, would probably be good to use that here too and keep it uniform